src/lib/validation/utils/assignHandlerForTrigger.js

```diff
@@ -44,10 +45,19 @@ export default function assignHandlerForTrigger (triggerButton, form) {
       invalidCustomErrorFields.forEach(field => {
         dplan.notify.notify('error', Translator.trans(field.validationMessage))
       })
-      // Make sure the default notification is displayed as needed
+      const nonEmptyFieldNames = [...new Set(
+        validatedForm.invalidFields
+          .map(field => field.getAttribute('data-dp-validate-error-fieldname'))
+          .filter(Boolean)
```

Comment:
what does this do? Looks for me, as if it always return true

Reply:
it return a list of fields. It extracts the value of the `data-dp-validate-error-fieldname` attribute from each element and filter empty/falsy values from the array.

Reply:
Didn't now that https://stackoverflow.com/questions/30016773/javascript-filter-true-booleans